### PR TITLE
Change token symbol size to 100

### LINKED ISSUE #1255: Change token symbol size to 100
**Problem**
HAPI is [changing](https://github.com/hashgraph/hedera-services/issues/757) the max symbol size from 32 to 100 characters. Our database will error if we receive a symbol larger than 32.

**Solution**
- Change the database field to varchar(100)
- Allow any UTF-8 characters. Ensure there's no logic that limits it to upper case or non whitespace.

**Alternatives**

**Additional Context**

### PR DESCRIPTION
**Pull request by xin-hedera**
**Detailed description**:

- Change symbol to varchar(100)

**Which issue(s) this PR fixes**:
Fixes #1255 

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: hedera-mirror-importer/src/main/resources/db/migration/V1.31.0__token_symbol_size_100.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+-- HAPI changes the max symbol size to 100 characters, apply the same to db table
```
Comment (Nana-EC) [inline hedera-mirror-importer/src/main/resources/db/migration/V1.31.0__token_symbol_size_100.sql:1]: nit:
```suggestion
-- HAPI changed the max symbol size to 100 characters, apply the same to db table
```

File: hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/migration/V1_31_1_fix_address_book_102Test.java [hunk 2]
```diff
@@ -36,15 +36,15 @@
 import com.hedera.mirror.importer.domain.EntityTypeEnum;
 import com.hedera.mirror.importer.repository.AddressBookRepository;
 
-public class V1_31_0_fix_address_book_102Test extends IntegrationTest {
+public class V1_31_1_fix_address_book_102Test extends IntegrationTest {
```
Comment (Nana-EC) [inline hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/migration/V1_31_1_fix_address_book_102Test.java:39]: Why the rename of the sql migration?

**Review by Nana-EC — CHANGES_REQUESTED** [review]
I don't think you can rename the existing sql migration as the checksum will break dev deployments.
You should just take the next number and we'll see who gets in first and who needs to rename to the next available `major.minor.build` version

**Review by steven-sheehy — APPROVED** [review]

**Review by ijungmann — APPROVED** [review]

**Review by Nana-EC — APPROVED** [review]
LGTM

**Pull request merged**